### PR TITLE
feat(back): Add user validation by email

### PR DESCRIPTION
**Pull request by Arnaud-Adon**
## What does this PR do?

Add user validation by email

## Ticket trello

<https://trello.com/c/QK037SvA/16-add-user-validation-email>

## Tasks

- [x] Add a container mailed to docker to test an email sending when a user created, Add Nodemailer package to nodeJs, create template user validation
- [x] Add condition in function verifToken to token.js to detect req.user.token for url user token

## Screenshots
Capture d’écran 2019-04-14 à 22 55 36

Capture d’écran 2019-04-14 à 22 56 20

Capture d’écran 2019-04-14 à 22 56 40

### CODE REVIEW

File: utils/notifier/user-notifier.js [hunk 1]
```diff
@@ -0,0 +1,6 @@
+const Mailer = require('../../mailer/mailer');
+const MailTemplate = require('./user-notifier-mail-content');
```
Comment (so-mouga) [inline utils/notifier/user-notifier.js]: Why not use a twig to generate the template ?

File: utils/notifier/user-notifier.js [hunk 2]
```diff
@@ -0,0 +1,6 @@
...
+
+exports.mailNotifier = (user, token) => {
```
Comment (so-mouga) [inline utils/notifier/user-notifier.js]: the name function is maybe confused right ? mailNotifier doesn't make sense with the logic code, Maybe SendEmailValidationAccount or something like this ?

File: utils/notifier/user-notifier.js (hunk 1 above)
Comment (Arnaud-Adon) [reply utils/notifier/user-notifier.js]: we can try to use a twig template.

File: utils/notifier/user-notifier.js (hunk 2 above)
Comment (Arnaud-Adon) [reply utils/notifier/user-notifier.js]: Ok, i remove this for another name

**Pull request merged**